### DISCUSSION TITLE
Fsl3 with xDrip+

### DISCUSSION THREAD

**Discussion by KooistraT** [NightscoutFoundation/xDrip#2148] [Q&A]
Is it possible to use FSL3 with xDrip+? 

I know FSL3 is a CGM, but with xDrip+ I can calibrate and get a widget. 

Can anyone tell me how I can install it?

**Comment by yoshimo** [comment]
https://github.com/NightscoutFoundation/xDrip/issues/1474
Not directly yet, but there are solutions via LibreLinkUpload and a module for rooted devices.
